app/services/geo_concerns/discovery/document_builder/date_builder.rb

```diff
@@ -28,9 +29,15 @@ def layer_year
           end
 
           # Returns the date the work was modified.
-          # @return [String] date in rfc3339 format.
+          # @return [String] date in XMLSchema format.
           def date_modified
-            DateTime.rfc3339(geo_concern.solr_document[:date_modified_dtsi]).to_s
+            DateTime.rfc3339(geo_concern.solr_document[:date_modified_dtsi]).utc.xmlschema
```

Comment:
the `date_modified` comes from curation concerns (a system date) and `issued` is user input. I will add some rescue blocks to handle different input formats for `issued`